python/pyspark/ml/clustering.py

```diff
@@ -88,6 +88,14 @@ def clusterSizes(self):
         """
         return self._call_java("clusterSizes")
 
+    @property
```

Comment:
Why is this now added, by mistake? this change should just be the one above and the test change